package test_proto_msg;

message Person {   
	required int32 id = 1;   
	required string name = 2;   
	optional string email = 3; 
	required uint32 data1 = 4;
	required uint32 data2 = 5;
}